### PR TITLE
More comprehensible examples

### PR DESCRIPTION
**Pull request by sbrunner**
Proper naming, no unneeded configuration.

### CODE REVIEW

File: core/src/test/resources/org/mapfish/print/processor/map/image_layer_test/config.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 templates:
   main: !template
-    reportTemplate: 'dummy.jxml'
+    reportTemplate: 'dummy.jrxml'
```
Comment (sebr72) [inline core/src/test/resources/org/mapfish/print/processor/map/image_layer_test/config.yaml:3]: why to change this one as jxml is used everywhere else?

File: core/src/test/resources/org/mapfish/print/processor/map/image_layer_test/config.yaml [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
     attributes:
-      mapDef: !map
+      map: !map
```
Comment (sebr72) [inline core/src/test/resources/org/mapfish/print/processor/map/image_layer_test/config.yaml:5]: Was mapDef used to test the none default map attribute ?

File: core/src/test/resources/org/mapfish/print/processor/map/image_layer_test/config.yaml (hunk 1 above)
Comment (sbrunner) [reply core/src/test/resources/org/mapfish/print/processor/map/image_layer_test/config.yaml:3]: `jxml` is used nowhere (expect here)!

File: core/src/test/resources/org/mapfish/print/processor/map/image_layer_test/config.yaml (hunk 2 above)
Comment (sbrunner) [reply core/src/test/resources/org/mapfish/print/processor/map/image_layer_test/config.yaml:5]: I don't understand the question!

File: core/src/test/resources/org/mapfish/print/processor/map/image_layer_test/config.yaml (hunk 1 above)
Comment (sebr72) [reply core/src/test/resources/org/mapfish/print/processor/map/image_layer_test/config.yaml:3]: grep -r 'dummy.jxml' ./ | wc = 106 occurences

Comment (sbrunner) [reply core/src/test/resources/org/mapfish/print/processor/map/image_layer_test/config.yaml:3]: Oups, fixed :-)

File: core/src/test/resources/org/mapfish/print/processor/map/image_layer_test/config.yaml (hunk 2 above)
Comment (sbrunner) [reply core/src/test/resources/org/mapfish/print/processor/map/image_layer_test/config.yaml:5]: The `inputMapper`/`outputMapper` are well tested in the `multiple_maps` and `oereb` example, no need to be tested here.
We will keep the example as simple as possible.

**Review by sebr72 — APPROVED** [review]
Looks good. Thanks

**Pull request merged**